app/src/main/java/org/fossasia/openevent/general/favorite/FavoriteEventsRecyclerAdapter.kt

```diff
@@ -9,10 +9,10 @@ import kotlinx.android.synthetic.main.item_card_favorite_event.view.favoriteFab
 import org.fossasia.openevent.general.R
 import org.fossasia.openevent.general.event.Event
 import org.fossasia.openevent.general.common.EventClickListener
-import org.fossasia.openevent.general.event.EventViewHolder
 import org.fossasia.openevent.general.common.EventsDiffCallback
 import org.fossasia.openevent.general.common.FavoriteFabClickListener
 import org.fossasia.openevent.general.common.ShareFabClickListener
+import org.fossasia.openevent.general.event.EventViewHolder
```

Comment:
Why these changes?

Reply:
Made a mistake while resolving conflicts will resolve this

Reply:
Resolved